src/pages/Typing/index.tsx

```diff
@@ -20,35 +18,22 @@ import usePronunciation from './hooks/usePronunciation'
 import Tooltip from 'components/Tooltip'
 import { useRandomState } from 'store/AppState'
 import Progress from './Progress'
+import { ResultScreen, IncorrectInfo, ResultSpeedInfo } from 'components/ResultScreen'
```

Comment:
Three components in a single file? Please put them in three files separately.

Reply:
Kaiyi made this adjustment and I guess maybe it's because only ResultScreen is the component, the two Infos are prop types from ResultScreen.tsx ?